.web.yml

```diff
@@ -0,0 +1,25 @@
+tutorials:
+  - title: Snorkel Intro Tutorial
+    description: Labeling data for spam classification
+    notebook: spam/01_spam_tutorial.ipynb
+  - title: Introduction to Transformation Functions
+    description: Data augmentation for spam classification
+    notebook: spam/02_spam_data_augmentation_tutorial.ipynb
+  - title: Introduction to Slicing Functions
+    description: Monitoring models for spam classification
+    notebook: spam/03_spam_data_slicing_tutorial.ipynb
+  - title: Hybrid Crowd Labeling Workflows in Snorkel
+    description: Programmatic and crowdworker labels for sentiment analysis
+    notebook: crowdsourcing
```

Comment:
why are some entries for `notebook` are formatted as `*.ipynb` and others aren't?

Reply:
if only one notebook in directory (most cases), find it automatically

Reply:
nit but can we have the .ipynb everywhere so it's easier to pattern match